kuma/static/styles/components/wiki/indicators.scss

```diff
@@ -356,13 +433,15 @@ Special snowflakes
     @extend %indicator-link-invert;
 }
 
-/* harmony */
-.overheadIndicator[style='background: #9CF49C;'],
-.overheadIndicator[style='background: rgb(156, 244, 156);'] {
-    border-color: #{map-get( map-get($colors-lookup, 'positive'), 'medium')} !important; /* stylelint-disable-line declaration-no-important */
-    background: #{map-get( map-get($colors-lookup, 'positive'), 'light')} !important; /* stylelint-disable-line declaration-no-important */
+.template-jsOverrides {
+    border-color: #d1d1ff;
+    background: #f5f5ff;
 }
 
+/* es7 and harmony macros */
+.esHarmony {
+    @include box-theme('positive');
```

Comment:
I feel like we need to be consistent with the colors we assign to these callout widgets. Experimental feel to me more like a blue, or even orange, than a green.

Here for example it is blue:
http://localhost:8000/en-US/docs/Web/API/DeviceLightEvent/Using_light_sensors

But then on the `Object.observe` page locally it is green:
http://localhost:8000/tr/docs/Web/JavaScript/Reference/Global_Objects/Object/observe

I would suggest that we keep this consistent, and rather opt for blue in experimental cases.

Reply:
I think I’ll just use the `.experimental` and `.indicator-warning` classes for `{{ES7}}` to match `{{SeeCompatTable}}`.